libs/libc/stdio/lib_libfread_unlocked.c

```diff
@@ -46,13 +46,14 @@
 
 ssize_t lib_fread_unlocked(FAR void *ptr, size_t count, FAR FILE *stream)
 {
-  FAR unsigned char *dest = (FAR unsigned char *)ptr;
+  FAR char *dest = (FAR char *)ptr;
   ssize_t bytes_read;
   size_t remaining = count;
 #ifndef CONFIG_STDIO_DISABLE_BUFFERING
   size_t gulp_size;
   int ret;
 #endif
+  int fd = (int)(intptr_t)stream->fs_cookie;
```

Comment:
need move to line 72 since we can known stream 
isn't NULL only after line 60

Reply:
Done